worlds/stardew_valley/options/forced_options.py

```diff
@@ -35,6 +36,16 @@ def force_walnutsanity_deactivation_when_ginger_island_is_excluded(world_options
                        f"Ginger Island was excluded from {player} ({player_name})'s world, so walnutsanity was force disabled")
 
 
+def force_qi_special_orders_deactivation_when_ginger_island_is_excluded(world_options: options.StardewValleyOptions, player: int, player_name: str):
+    ginger_island_is_excluded = world_options.exclude_ginger_island == options.ExcludeGingerIsland.option_true
+    qi_board_is_active = world_options.special_order_locations.value & options.SpecialOrderLocations.value_qi
+
+    if ginger_island_is_excluded and qi_board_is_active:
+        world_options.special_order_locations.value -= options.SpecialOrderLocations.value_qi
+        logger.warning(f"Mr. Qi's Special Orders requires Ginger Island. "
```

Comment:
Could we maybe log the chosen value, and the resulting value?

Reply:
![image](https://github.com/user-attachments/assets/29bd5e36-8ba0-4479-8d22-2d9f15f0a41e)